scss/_feedback.scss

```diff
@@ -0,0 +1,125 @@
+.slider {
+  width: 100%;
+  margin: 0 auto;
+}
+
+.carousel-item {
+  width: 100%;
+  height: auto;
+
+  .item {
+    width: 100%;
+    display: inline-block;
+  }
+}
+
+.item {
+  padding: 0 5px;
+  .comment {
+    background: $white_color;
+    border: 0.5px solid $color_border;
+    box-sizing: border-box;
+    border-radius: 10px;
+    padding: 4% 2%;
+    height: auto;
+    display: flex;
+    box-shadow: 0px 0px 10px $color_box;
+  }
+
+  span.line {
+    display: block;
+    height: auto;
+    width: 10px;
+    background-color: $color_border;
+    margin: 0 15px;
+  }
+}
+
+.info {
+  margin-top: 10px;
+  display: flex;
+  padding: 5px 10px;
+
+  .avt-author {
+    padding: 5px;
+  }
+
+  .data {
+    margin-left: 10px;
+
+    span {
```

Comment:
không nên select vào thẻ như này

Reply:
cái này mình css cho toàn bộ thẻ span trong div data

Reply:
cái này mình css cho toàn bộ thẻ span trong div data

Reply:
à ok b, lần sau tôi sẽ check kĩ hơn